test/client/fixtures/sandbox/node/attributes-test.js

```diff
@@ -122,8 +120,7 @@ test('url', function () {
         { tagName: 'script', attr: 'src' },
         { tagName: 'link', attr: 'href' },
         { tagName: 'form', attr: 'action' },
-        { tagName: 'embed', attr: 'src' },
-        { tagName: 'object', attr: 'data' }
+        { tagName: 'embed', attr: 'src' }
     ];
 
     if (!browserUtils.isIE || browserUtils.version > 9) {
```

Comment:
Remove this condition here and in all places (It is for IE9+)

Reply:
What about it?

Reply:
https://github.com/DevExpress/testcafe-hammerhead/pull/1479/files#diff-4b08ab790386be4d04295aa29c8be49eL129

Reply:
Ok